x-pack/plugins/apm/server/lib/transactions/charts/get_timeseries_data/transform.ts

```diff
@@ -64,6 +64,11 @@ export function getTpmBuckets(
       };
     });
 
+    // Handle "missing" result keys
+    if (key === '') {
+      key = 'N/A';
+    }
+
```

Comment:
1) Have you given any thought to how `null` values should be treated? Normally buckets with `key=null` are stripped out but we include them under the key "transaction_result_missing":
https://github.com/elastic/kibana/blob/50ec75f800b91172c877b5aa0d6e446bb09780fb/x-pack/plugins/apm/server/lib/transactions/charts/get_timeseries_data/fetcher.ts#L142

Perhaps this logic should be kept together in the backend?

2) Localization ;)

3) Can we avoid the re-assignment and keep `key` immutable?
There is actually a tslint rule for this, that I think would make sense to enable `"no-parameter-reassignment": true`.

Reply:
>Perhaps this logic should be kept together in the backend?
Thoughts on how to do that besides changing the "missing" value to match this new value?

> Localization
Good catch, I see we have a localized `NOT_AVAILABLE_MESSAGE` in `public/constants/index.ts` that we can move to somewhere in `common` so it can be shared by server code.

>Can we avoid the re-assignment and keep key immutable?
I'm not as zealous about immutability as you are, I think, but if it makes you happy :)